packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -105,23 +122,23 @@ const defaultProps = {
 function WorklistDataGrid(props) {
   const {
     id,
-    ariaLabelledby,
     ariaLabel,
+    ariaLabelledBy,
     columns,
-    rows,
```

Comment:
I was trying to understand the purpose of reordering the props.  It is not alphabetical.

Reply:
We basically tried to minimize finding a prop all over the list. The reason we didn't alphabetize them is having the various row/column etc props be next to each other made more logical sense.

Reply:
We are actively changing these props at the moment.  Maybe this should be something we delay until towards the end of the project.  I don't think the changes provide much benefit at the moment.

Reply:
I see your edit and I agree. The main advantage of reordering them is so that the props table is more organized on the dev site but now that i think about it, it can be done as part of the documentation jira when the main pieces are in place.

Creating a new PR that doesn't have the prop reordering
#1702